### DISCUSSION TITLE
How to write xls file synchronously

### DISCUSSION THREAD

**Discussion by KalaiselvanMahendran** [exceljs/exceljs#2511] [Q&A]
I want to write xlsx file synchronously, Is there any method to write below in synchronous way

From this to ,
```
workbook.xlsx.writeFile(fileName).then(function(){
    console.log("Writing completed..");
});
```

Like this
`workbook.xlsx.writeFileSync(fileName);`

or some other method ?

**Comment by mblackcat** [comment]
mark
